aframe/tasks/plots/sv.py

```diff
@@ -9,11 +10,27 @@
 from aframe.tasks.infer import InferLocal
 
 
+def load_prior(path):
+    """
+    Load prior from python path string
+    """
+    module_path, prior = path.rsplit(".", 1)
+    module = importlib.import_module(module_path)
+    prior = getattr(module, prior)
+    return prior
+
+
 class SensitiveVolume(AframeSingularityTask):
     """
     Compute and plot the sensitive volume of an aframe analysis
     """
 
+    mass_combos = luigi.ListParameter(
+        description="Mass combinations for which to calculate sensitive volume"
+    )
+    source_prior = luigi.Parameter(
+        "Python path to prior to use for waveform generation"
```

Comment:
Maybe a description like "Python path to prior that was used for generating testing waveform injections" will be a bit more clear